src/services/cache-config/CacheConfigService.ts

```diff
@@ -387,6 +398,17 @@ export class CacheConfigService {
     return this.dmrService.executePost(builder.build());
   }
 
+  private getMemoryType(memory: any): string {
+    let memoryType: string = "";
+    for (var fieldName in memory) {
+      // there should be only one object BINARY, OBJECT, or OFF-HEAP
+      if (isObject(memory[fieldName])) {
+        memoryType = fieldName;
```

Comment:
If there is only one object, can we not just return `fieldName` here and remove the `memoryType` variable?